dlang/source/flatbuffers/bytebuffer.d

```diff
@@ -0,0 +1,85 @@
+module flatbuffers.bytebuffer;
+
+import std.exception;
+import std.bitmanip;
+import std.exception;
+import core.exception;
+
+final class ByteBuffer
+{
+public: 
+	this(ubyte[] buffer)
+	{
+		_buffer = buffer;
+		_pos = 0;
+		_bigEndian = false; //faltbuffer 在（x86下）默认小端存储
```

Comment:
flatbuffer is used he endianness of the current machine? in x86_64 we test is little endian.